caffe2/core/macros.h.in

```diff
@@ -25,6 +25,7 @@ static_assert(
 
 #cmakedefine CAFFE2_BUILD_SHARED_LIBS
 #cmakedefine CAFFE2_FORCE_FALLBACK_CUDA_MPI
+#cmakedefine CAFFE2_USE_CUDA_MPI
```

Comment:
Can you please elaborate why downstream libraries need to be aware of this define? I.e. does it introduce some sort of new API that libtorch C++ developers can now use
If not, perhaps it can be added as compile option for a specific .cpp file?